scratchspace/SphereScat/testdriver4f90.f

```diff
@@ -356,7 +356,9 @@
      1    centers,nterms,sh1,sh2,targ,ntargs,htot,
      2    fld,cw,lcw,lused,ier)
       open(unit=20, file='data.m',status='unknown') 
-      open(unit=21, file='field.m',status='unknown') 
+      open(unit=21, file='field.m',status='unknown')
+      open(unit=22, file='data.bin',status='unknown')

```

Comment:
@alisonsimons, should we contact Leslie to get a hint on how to write the .bin data in fortran?

Reply:
@magland Yeah seems like a good idea